understanding/compatible.html

```diff
@@ -54,49 +52,39 @@
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#success-criteria">Success Criteria</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#success-criteria">達成基準</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				Guideline 4.1:</span>Compatible
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				ガイドライン 4.1:</span>互換性
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Guideline </header>
+               <header class="box-h  box-h-icon">ガイドライン</header>
                <div class="box-i">
-                  <p>Maximize compatibility with current and future user agents, including assistive technologies.</p>
+                  <p>現在及び将来の、支援技術を含むユーザエージェントとの互換性を最大化すること。</p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this guideline is to support compatibility with current and future
-                  user agents, 
-                  <em>especially</em> assistive technologies (AT). This is done both by 1) ensuring that authors do not
-                  do things that would break AT (e.g., poorly formed markup) or circumvent AT (e.g.,
-                  by using unconventional markup or code) and 2) exposing information in the content
-                  in standard ways that assistive technologies can recognize and interact with. Since
-                  technologies change quickly, and AT developers have much trouble keeping up with rapidly
-                  changing technologies, it is important that content follow conventions and be compatible
-                  with APIs so that AT can more easily work with new technologies as they evolve.
+               <h2>意図</h2>
+               <p>このガイドラインの目的は、現在及び将来のユーザエージェントの互換性、<em>特に</em>支援技術との互換性をサポートすることである。これは、1) コンテンツ制作者が支援技術の機能を損なったり (例: 形式が整えられていないマークアップ)、又は支援技術を回避したり (例: 奇抜なマークアップ又はソースコードを用いる) しないことを保証する、2) 支援技術が認識して情報のやりとりができる標準的な方法でコンテンツにある情報を公開することの両方で達成できる。技術の変化は早く、支援技術の開発者が急速に変化する技術に追従していくことは大きな困難を伴うので、支援技術が容易に新しい技術にも対応していけるように、コンテンツが規則に従っていて API との互換性が確保されていることが重要である。
```

Comment:
ダメではないのですがなんかこの辺り全体的にわかりにくい感じがしました。

まず「形式が整えられていないマークアップ」というのがなんなのかよくわからないなぁという感想を持ちました。原文 poorly formed ですが、これは well-formed の反対というニュアンスで、今は亡き 4.1.1 を意識したものでしょう。その意図を汲んで「整形式でないマークアップ」としてしまったほうが良いのかもしれません。

「支援技術を回避」もよくわからない感じがします。circumventは「出し抜く」とか「騙して逃れる」というようなネガティブなニュアンスですが、回避だと中立な感じがするというのもあります。

「奇抜なマークアップ」というのもよくわからない感じです。原文は unconventional で、これは「慣習に従わない」「標準に従わない」というような意味かと思います。下では convention を「規則」としていますので、そこと合わせるのが良いと思います。

```suggestion
               <p>このガイドラインの目的は、現在及び将来のユーザエージェントの互換性、<em>特に</em>支援技術との互換性をサポートすることである。これは、1) コンテンツ制作者が支援技術の機能を損なったり (例: 整形式でないマークアップ) 又は支援技術を出し抜いたり (例: 規則に従わないマークアップ又はコードの使用) しないことを保証すること、2) 支援技術が認識して情報のやりとりができる標準的な方法でコンテンツにある情報を公開すること、の両方で達成できる。技術の変化は早く、支援技術の開発者が急速に変化する技術に追従していくことは大きな困難を伴うため、支援技術が容易に新しい技術にも対応していけるように、コンテンツが規則に従っていて API との互換性が確保されていることが重要である。
```